common/api/core-geometry.api.md

```diff
@@ -1495,15 +1495,13 @@ export abstract class CurveChain extends CurveCollection {
     protected _curves: CurvePrimitive[];
     cyclicCurvePrimitive(index: number, cyclic?: boolean): CurvePrimitive | undefined;
     endPoint(result?: Point3d): Point3d | undefined;
-    endPointAndDerivative(result?: Ray3d): Ray3d | undefined;
```

Comment:
Looks like you don't have the latest changes?  These changes [merged](https://github.com/iTwin/itwinjs-core/pull/7445) 4 days ago.